CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ _None_
 * Update the Swift version in `.swift-version` so that the right version is used when building manually (using `swiftenv`).  
   [@cfiken](https://github.com/cfiken)
   [#764](https://github.com/SwiftGen/SwiftGen/issues/764)
+* Update Yams version from `3.0.0` to `4.0.0` in order to use latest version of SwiftGen with other popular tools (such as SwiftLint) as Swift package dependencies.  
```

Comment:
This belongs under the "Internal changes", there's no real need to explain "why" we upgraded.

Reply:
Makes sense, moved it.